libcloud.groovy

```diff
@@ -27,32 +27,42 @@ def replicate_to_clouds(pipecfg, basearch, buildID, stream) {
             }
         }
     }
+
+    // For AWS we need to consider the primary AWS partition and the
+    // GovCloud partition. Define a closure here that we'll call for both.
+    def awsReplicateClosure = { config, credentialId ->
+        def creds = [file(variable: "AWS_CONFIG_FILE", credentialsId: credentialId)]
+        withCredentials(creds) {
+            def c = config
+            shwrap("""
+            cosa aws-replicate \
+                --log-level=INFO \
+                --build=${buildID} \
+                --arch=${basearch} \
+                --credentials-file=\${AWS_CONFIG_FILE} \
```

Comment:
Extra \ here

Reply:
ok fixed that and added one more commit!